db/migrate_cards/data/cards/source_structure

```diff
@@ -0,0 +1,32 @@
+
+<div class="col-md-12 column">
+  <div class="row clearfix">
+    {{_|listing}}
+  </div>
+	<div class="row clearfix">
+		<div class="col-1 column">
+		  <i class="fa fa-info-circle"></i>
+		</div>
+		<div class="col-md-11 column">
+		  {{+Title|titled;title:Title}}
+		  {{+Description|titled;title:Description}}
+		  {{+Year|titled;title:Year|name}}
+		  {{#+Date|titled;title:Date}}
+		  {{+Report type|titled;title:Report Type|name}}
+      {{#+Website|titled;title:Website|link}}
+		</div>
+	</div>
+	<div class="row clearfix">
+		<div class="col-1 column">
+		  <i class="fa fa-tag"></i>
+		</div>
+		<div class="col-md-11 column">
+		    {{+Companies|titled; title:Companies|link}}
+        {{+Topics|titled; title:Topics|link}}
+        {{#+Tags|titled;title:Tags; item:link}}
+		</div>
+	</div>
+	<div class="row clearfix">
+	  {{+Discussion|titled;title:Discussion;show:comment_box,menu}}
+	</div>
+</div>
```

Comment:
hound can't recognize that but html structures in cards is considered to be bad style. I suggest you make it a view and convert it to haml (e.g with http://htmltohaml.com) and replace nests like ` {{+Year|titled;title:Year|name}}` with `= field_nest :nest, view: :titled, title: "Year", items: { view: :name}`